### PR TITLE
[16.0][MIG] sale_procurement_group_by_line: Migration to 16.0

### PR DESCRIPTION
**Pull request by kimirondon**

### CODE REVIEW

File: sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        )
+        self.line2 = self.order_line_model.create(
+            {
+                "order_id": self.sale.id,
+                "product_id": self.new_product2.id,
+                "product_uom_qty": 5.0,
+                "name": "Sale Order Line Demo2",
+            }
+        )
+        return self.sale
```
Comment (BT-anieto) [inline sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py]: As suggestion, I would convert it to setupcls and classmethods to speed up the tests

**Review by BT-anieto — APPROVED** [review]

**Comment by BT-anieto** [comment]
@kimirondon Hi! Thank you very much for migrating this module. Could you please?

1) Change the PR name (Add the missing '[' to MIG])
2) Squash the commits as explained in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

Thx again! :)

**Comment by BT-anieto** [comment]
Hi @kimirondon! :) Are you going to continue working of that or shall another dev take it?

**Comment by kimirondon** [comment]
hellou @BT-anieto !
yes, I will continue working on this PR, sorry for my delay.

**Review by Fabiola-Auz — APPROVED** [review]

**Comment by kimirondon** [comment]
hellou @BT-anieto! done, I squash the commit history.

File: sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+# © 2016-20 ForgeFlow S.L. (https://www.forgeflow.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleProcurementGroupByLine(TransactionCase):
+    def setUp(self):
```
Comment (hugo-cordoba) [inline sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py]: Hi, as @BT-anieto  says, can you change it to SetUpClass?

Comment (kimirondon) [reply sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py]: hello @HugoCordobaLeal !! done, i updated the unit tests.
@BT-anieto FYI

**Pull request merged**